app/src/main/java/com/practicum/playlistmaker/di/ReposiroryModule.kt

```diff
@@ -8,7 +11,13 @@ import org.koin.dsl.module
 val reposiroryModule = module{
 
     single<TrackRepository> {
-        TrackRepositoryImpl(get(), androidContext())
+        TrackRepositoryImpl(networkClient = get(), context = androidContext(), appDataBase = get())
```

Comment:
```suggestion
        TrackRepositoryImpl(
            networkClient = get(), 
            context = androidContext(), 
            appDataBase = get()
        )
```

Обычно если параметров больше 1 - их пишут каждый с новой строки